### DISCUSSION TITLE
Add new features like order tracking and inventory management

### DISCUSSION THREAD

**Discussion by waelmansourx** [twentyhq/twenty#1779] [Ideas]
Here are a list of great features to include, inspired from CRMs like Zoho and Hubspot:

- Contact management
- Order tracking
- Inventory management
- Sales pipeline visualization
- Communication tools
- Reports & analytics

**Comment by FelixMalfait** [comment]
Haha giving us a lot of work there @waelmansourx :) 

Long-term we want to allow people to create custom module/plugins so the community could definitely build an inventory management feature for example. But we're not there just yet
